src/server/routes/unsupportedBrowserRouter.tsx

```diff
@@ -49,6 +49,17 @@ const unsupportedBrowserRouter = (_: Request, res: Response) => {
       <title>Unsupported browser</title>
       <meta name="description" content="Your browser is not supported"></meta>
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <script>
+      if('${process.env.REPORT_ANALYTICS}'.toLowerCase() === 'true'){
```

Comment:
We could go one step further and make the whole addition of the script to the payload conditional on `process.env.REPORT_ANALYTICS`.